### PR TITLE
feat(identity): enable browser caching

### LINKED ISSUE #257: feat(identity): browser caching
Consider setting a `Cache-Control` header so that identity responses can be cached locally for a time.

What's a good value for this? 1 minute? 1 hour?

This is different from the Redis cache and cache invalidation from blockchain events which is tracked here: https://github.com/WalletConnect/rpc-relay/issues/6

1 hour may be too long, especially if used for transactions. 1 minute may be sufficient as an easy win. The identity API is pretty fast already so this is not important.

### PR DESCRIPTION
**Pull request by chris13524**
# Description

Adds `Cache-Control` header with a value that matches the current TTL of the Redis cache.

Resolves #257 

## How Has This Been Tested?

New unit tests

## Due Diligence

* [ ] Breaking change
* [ ] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

**Review by arein — APPROVED** [review]
LGTM - consider also covering in integration tests

File: src/handlers/identity.rs [hunk 1]
```diff
@@ -239,9 +258,9 @@ async fn lookup_identity(
             debug!("Saving to cache");
             let cache = cache.clone();
             let res = res.clone();
-            let cache_ttl = Duration::from_secs(60 * 60 * 24);
             // Do not block on cache write.
             tokio::spawn(async move {
+                let cache_ttl = CACHE_TTL.to_std().expect("valid duration");
```
Comment (geekbrother) [inline src/handlers/identity.rs:263]: Probably `invalid duration` not valid?

**Review by geekbrother — APPROVED** [review]
lgtm, thanks!

File: src/handlers/identity.rs (hunk 1 above)
Comment (chris13524) [reply src/handlers/identity.rs:263]: Thanks, fixed in 33c224e67baf209a785815417d499038806b57c4

**Pull request merged**